doc/nrf/ug_dev_model.rst

```diff
@@ -194,7 +194,7 @@ If you already have a copy of the |NCS| and wish to update it or switch to a new
   west update
 
 Where ``{remote}`` is the Git remote that points to the official Nordic repository.
-Typically, this is called ``origin`` but may have been renamed.
+This is called ``origin`` by default for the `fw-nrfconnect-nrf`_ repository and ``ncs`` for most others, but :ref:`may have another name <dm-wf-fork>`.
```

Comment:
How are the repository and the name of the remote related? I don't get why you say that for one repository the remote is called something and for another repository it's called something else.

Reply:
Because the manifest repo remote is always called `origin` by default, whereas the remote for all the projects in the manifest is named after the `remote` section in `west.yml`. In our case, `ncs`.

Reply:
ok, thanks.